### PR TITLE
Update mcrl2

### PR DESCRIPTION
**Pull request by moretea**
###### Things done
- [X] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"` **Nothing depends on it**
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Hi there, this is my first attempt to contribute to nixpkgs. I packaged mcrl2, but discovered that a much older version was already packaged. 

Furthermore, I think that  location of `pkgs/applications/science/logic` is more appropriate than `pkgs/tools/misc/`.

If you agree with these changes, I'll squash the commits into one commit with the description `mcrl2: 2009.01 -> 2014.09`.

### CODE REVIEW

File: pkgs/applications/science/logic/mcrl2/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  buildInputs = [ xlibs.libX11 cmake subversion mesa qt5.qtbase boost
+                  python27 python27Packages.pyyaml python27Packages.psutil ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "A formal specification language with an associated toolset, " +
+                  "that can be used for modelling, validation and verification of " +
+                  "concurrent systems and protocols.";
```
Comment (zimbatm) [inline pkgs/applications/science/logic/mcrl2/default.nix]: Is there no shorted description of the package ? It should really fit on a single line to be useful in tools like `nox`.

There is a longDescription attribute available if you want to expand further on what the package does.

**Comment by zimbatm** [comment]
:+1: let me know when it's squashed

**Comment by moretea** [comment]
@zimbatm ping

File: pkgs/applications/science/logic/mcrl2/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  buildInputs = [ xlibs.libX11 cmake subversion mesa qt5.qtbase boost
+                  python27 python27Packages.pyyaml python27Packages.psutil ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    description = "A toolset for model-checking concurrent systems and protocols";
+    longDescription = "A formal specification language with an associated toolset, " +
+                      "that can be used for modelling, validation and verification of " +
+                      "concurrent systems and protocols";
```
Comment (zimbatm) [inline pkgs/applications/science/logic/mcrl2/default.nix]: Sorry, the longDescription should use the multi-line double-single quote format.

**Comment by zimbatm** [comment]
@moretea a last fix and then it's good to go

**Comment by moretea** [comment]
@joachifm thanks!

**Pull request merged**